src/algebra/big_operators/basic.lean

```diff
@@ -570,6 +570,14 @@ by { rw prod_ite, simp [filter_false_of_mem h, filter_true_of_mem h] }
   (h : ∀ x ∈ s, p x) : (∏ x in s, if p x then f x else g x) = (∏ x in s, f x) :=
 by { simp_rw ←(ite_not (p _)), apply prod_ite_of_false, simpa }
 
+@[to_additive] lemma prod_apply_ite_of_false {p : α → Prop} {hp : decidable_pred p} (f g : α → γ)
```

Comment:
The reason I proved these in this way, as opposed to proving these and specializing, was because Lean was not having a fun time coercing `filter_{false/true}_of_mem h` into the right form.